### DISCUSSION TITLE
Quick search within collection pane

### DISCUSSION THREAD

**Discussion by engdan77** [darrenburns/posting#124] [Ideas]
Awesome project, I must say... Great job !!! 🥇 

Something that I believe would be found very valuable would be if there were a quick "search" within a collection (e.g., CTRL-F) that could be used to enter a word that would take you to the first request where it matches and then also support, e.g., CTRL-N/CTRL-P or similar for the jump between multiple matching requests.

**Comment by darrenburns** [comment]
I've added a feature which I think covers this in 2.4.0.

You can press `ctrl+shift+p` to open a fuzzy search popup, allowing you to quickly move between requests.
